scvi/core/models/__init__.py

```diff
@@ -620,7 +724,7 @@ def get_latent_library_size(
 
 
 class BaseModelClass(ABC):
```

Comment:
Same here, this should be commented so we know what this is doing

Reply:
same